web/styles/popovers.css

```diff
@@ -617,21 +617,28 @@ ul.popover-group-menu-member-list {
     .stream-list-top-section {
         display: flex;
         margin-top: 12px;
+        align-items: center;
+        margin-bottom: 5px;
 
-        .stream-search {
+        .stream-search-container {
             margin-left: auto;
-            align-self: center;
+            display: grid;
+            grid-template: "search-input clear-search" auto / minmax(0, 1fr) 30px;
```

Comment:
Not sure of the name here since the "x" button is eventually inside the input only.